### ISSUE TITLE
FreeTime should be able to remember Timetables across restarts.

### ISSUE THREAD

**Issue by alexiscatnip** [CS2113-AY1819S1-W13-1/main#69]
Changes to any timetable exist only in model, and are not saved to disk. Thus, each time the application is restarted, the timetables are empty. 

I will work on saving the timetable and timeslots to disk, each time "AddressBookChangedEvent" is called.

**Comment by bennchong** [comment]
```    @Subscribe
    public void handleAddressBookChangedEvent(AddressBookChangedEvent event) {
        logger.info(LogsCenter.getEventHandlingLogMessage(event, "Local data changed, saving to file"));
        try {
            saveAddressBook(event.data);
        } catch (IOException e) {
            raise(new DataSavingExceptionEvent(e));
        }
    }```

The above code is from StorageManager, it seems like all you have to do is raise this event where ever you change the timetable? Or if you want to be more efficient whenever you close the app?

**Comment by alexiscatnip** [comment]
@bennchong yep, the main issue is that AB4 only saved the person's details that AB4 originally had: the name, phone,email, etc into the AddressBook.xml file. It did not save the timetable, timeslots.

**Comment by alexiscatnip** [comment]
Implemented in #76

**Comment by bennchong** [comment]
Good job fixing this
